CHANGES.md

```diff
@@ -1,3 +1,58 @@
+## Version 0.14.0
+
+This release is a major milestone for Spire.
+
+First of all, it introduces a dependency on typelevel/algebra (and
+thus on typelevel/cats-kernel). This provides immediate compatibility
+with Cats and Algebird.
+
+Second of all, Spire is now published to the org.typelevel
+organization (which was previously org.spire-math).
+
+Additions:
+ * Add GCDRing, generalized from EuclideanRing
+ * Add DivisionRing, a field without commutative multiplication
+
+Changes:
+ * Many type classes are now aliases to algebra's type classes
+ * The relationship between EuclideanRing and Gcd has changed
+ * IsReal is no longer required to be ordered, only signed
```

Comment:
To be precise: Signed now extends Order.

Also, Signed should be interpreted as a linearly ordered group, but we keep two typeclasses "type towers" to avoid proliferation of types/ambiguous implicits (also, Sign has a Signed instance without being a additive monoid).

However, we check the laws of linearly ordered groups.